### PR TITLE
Chaincode in user

### LINKED ISSUE #587: Change the way we are generating a chaincode
Currently, we just creating a document in chaincodes collection to create a chaincode
Now we also want to update it in the user's object.

We chaincode key in user's object just update that whenever we generate a new chaincode.

### PR DESCRIPTION
**Pull request by RahulGoyal-tech**
closes #587

### CODE REVIEW

File: test/utils/checkChaincode.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const users = require("../../models/users");
+module.exports = async (chaincode, userId) => {
+  const data = await users.fetchUser({ userId });
+  return data.user.chaincode === chaincode;
+};
```
Comment (pallabez) [inline test/utils/checkChaincode.js:5]: Nice :+1:

**Review by pallabez — APPROVED** [review]
LGTM :+1:

**Comment by RahulGoyal-tech** [comment]
@rohan09-raj @pallabez @lakshayman when will we be merging this PR as review is done by pallab and no conflicts are there

**Review by lakshayman — APPROVED** [review]
LGTM

**Pull request merged**